test/introspection-filter.js

```diff
@@ -9,11 +9,12 @@ const mercuriusAuth = require('..')
 const schema = `
   directive @auth on OBJECT | FIELD_DEFINITION
   directive @hasRole (role: String!) on OBJECT | FIELD_DEFINITION
-  directive @hasPermission (grant: String!) on OBJECT | FIELD_DEFINITION
+  directive @hasPermission (grant: String!) repeatable on OBJECT | FIELD_DEFINITION
```

Comment:
I think this option needs a test in the `gateway` feature too

Reply:
I've added a couple of tests cases for this